website/docs/r/is_vpc_routing_table.html.markdown

```diff
@@ -52,7 +67,7 @@ Review the argument references that you can specify for your resource.
 ## Attribute reference
 In addition to all argument reference list, you can access the following attribute reference after your resource is created.
 
-- `accept_routes_from_resource_type` - (List) The reource type filter specifying the resources that may create routes in this routing table.
+- `accept_routes_from_resource_type` - (List) The resource type filter specifying the resources that may create routes in this routing table. Ex: `vpn_server`, `vpn_gateway`
```

Comment:
if its in optional arguments, then it doesnt need to be in the exported attributes

Reply:
resolved.